src/App.tsx

```diff
@@ -21,34 +23,64 @@ function App() {
     getCurrencies()
   }, [])
 
+  useEffect(() => {
+      const setValueLatestPrice = async () => {
```

Comment:
Don't use value and price in one naming.
Price it's the wrong name for this variable, think about the more clear name

Reply:
+